src/polls/durability.md

```diff
@@ -10,3 +10,9 @@ Durability will cease to exist in the mod. Items will no longer break. Will affe
 Durability will cease to exist for all items from the WotRift mod. Vanilla items and items from other mods will still have durability.
 ### No, durability should stay
 Durability will remain in the mod. Items will still break.
+
+## Results
```

Comment:
Poll results would be great with a screenshot of the discord, as some sort of proof.